libexec/python/utils.py

```diff
@@ -57,13 +56,13 @@ def add_http(url,use_https=True):
     prefix = "https://"
     if use_https == False:
         prefix="http://"
-    
+
     # Does the url have http?
-    if re.search('^http*',url) == None:
+    if not url.startswith('http'):
```

Comment:
thanks @gmkurtzer - @flx42 let me mess around a bit and I'll come up with something that you like that fits the bill! Stay tuned :)